### ISSUE TITLE
compilation error

### ISSUE THREAD

**Issue by morozovsk** [tony2001/ngx_http_pinba_module#15]
[docker ](https://github.com/pinba-server/pinba-server/blob/master/docker/ubuntu18.04/Dockerfile)
ubuntu 18.04
nginx 1.14.0
gcc 7.3.0

```bash
apt install -y zlib1g-dev libgd-dev libxslt1-dev libgeoip-dev && \
    git clone https://github.com/tony2001/ngx_http_pinba_module && \
    wget http://nginx.org/download/nginx-1.14.0.tar.gz && tar zxvf nginx-1.14.0.tar.gz && cd nginx-1.14.0 && \
    ./configure --add-dynamic-module=/ngx_http_pinba_module --with-debug --with-pcre-jit --with-http_ssl_module --with-http_stub_status_module --with-http_realip_module --with-http_auth_request_module --with-http_v2_module --with-http_dav_module --with-http_slice_module --with-threads --with-http_addition_module --with-http_geoip_module=dynamic --with-http_gunzip_module --with-http_gzip_static_module --with-http_image_filter_module=dynamic --with-http_sub_module --with-http_xslt_module=dynamic --with-stream=dynamic --with-stream_ssl_module --with-mail=dynamic --with-mail_ssl_module && \
    make
```

errors:

> /ngx_http_pinba_module/ngx_http_pinba_module.c
> In file included from /ngx_http_pinba_module/ngx_http_pinba_module.c:9:0:
> /ngx_http_pinba_module/ngx_http_pinba_module.c: In function 'ngx_http_pinba_resolve_and_open_socket':
> /ngx_http_pinba_module/uthash.h:441:21: error: this statement may fall through [-Werror=implicit-fallthrough=]
>       case 11: hashv += ( (unsigned)_hj_key[10] << 24 );

> cc1: all warnings being treated as errors
> make[1]: *** [objs/addon/ngx_http_pinba_module/ngx_http_pinba_module.o] Error 1
> objs/Makefile:1531: recipe for target 'objs/addon/ngx_http_pinba_module/ngx_http_pinba_module.o' failed
> make[1]: Leaving directory '/nginx-1.14.0'
> make: *** [build] Error 2
> Makefile:8: recipe for target 'build' failed

**Comment by tony2001** [comment]
Please try compiling the latest master, I've updated uthash.h to the latest version and it should fix all compile-time warnings (at least that fixed it for me with GCC 8.3.1).

**Comment by morozovsk** [comment]
Now everything works fine.
Thanks.
